crates/ethcore/src/engines/authority_round/mod.rs

```diff
@@ -2025,10 +2033,22 @@ impl Engine<EthereumMachine> for AuthorityRound {
                             let chain_id = self.machine.params().chain_id;
 
                             if let Some(signer) = self.signer.read().as_ref() {
-                                let signature = signer.sign(t.signature_hash(Some(chain_id))).unwrap();
-                                let tx = SignedTransaction::new(t.with_signature(signature, Some(chain_id)))?;
-
-                                if let Err(e) = push_reward_transaction(&self.machine, block, tx, None) {
+                                let signature =
+                                    signer.sign(t.signature_hash(Some(chain_id))).unwrap();
+                                let tx = SignedTransaction::new(
+                                    t.with_signature(signature, Some(chain_id)),
+                                )?;
+
+                                trace!(target: "engine", "on_close_block: call self.current_fees_params");
+                                let fees_params = self.current_fees_params(&block.header);
```

Comment:
For the Reward transaction fee is None always.

Reply:
Resolved